addons/purchase/models/mail_compose_message.py

```diff
@@ -8,6 +8,4 @@ class MailComposeMessage(models.TransientModel):
     _inherit = 'mail.compose.message'
 
     def _action_send_mail(self, auto_commit=False):
```

Comment:
Can't we remove the override here ?

Reply:
Yeah, actually we can remove the whole file(s) since that's the only thing they do.

Done